scripts/generate_ipfs_hashes.py

```diff
@@ -206,7 +206,7 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         # terminate the ipfs daemon
-        self.process.send_signal(signal.SIGINT)
+        self.process.send_signal(signal.SIGTERM)
```

Comment:
is that cross-platform compatible then?

Reply:
yes. For some reason, `process.send_signal` does not accept `SIGINT`:

https://github.com/python/cpython/blob/3.8/Lib/subprocess.py#L1422-L1432